src/cbottle/inference.py

```diff
@@ -58,6 +58,22 @@ def _build_labels(labels, denoiser_when_nan: str):
     return out_labels
 
 
+def reset_torch_compile_cache():
```

Comment:
Please remove. It is torch's job to control its only cache, and we should not call into torch private apis like `torch._dynamo` . We can include this information in example scripts and documentation, but I don't think we should rebrand torch's private APIs.